zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -192,6 +192,14 @@ impl Config {
         cur: &Rw,
         prev: &Rw,
     ) -> Result<LimbIndex, Error> {
+        // annotate columns
```

Comment:
Same as comment to follow approach in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1167/files

Reply:
fixed in [1b5aa41](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/1b5aa4193380414a8378031399be7a6eddfac11a)